lib/ops/opsservice/endpoints.go

```diff
@@ -71,11 +71,11 @@ func (o *Operator) GetApplicationEndpoints(key ops.SiteKey) ([]ops.Endpoint, err
 	}
 
 	// query for nodes, we might need them later on
-	nodeList, err := client.Core().Nodes().List(metav1.ListOptions{})
+	nodeList, err := client.CoreV1().Nodes().List(metav1.ListOptions{})
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	namespaceList, err := client.Core().Namespaces().List(metav1.ListOptions{})
+	namespaceList, err := client.CoreV1().Namespaces().List(metav1.ListOptions{})
```

Comment:
I'm curious, why are we renaming all these clients to specific versions? Wouldn't that potentially increase the "refactoring burden" in the future?

Reply:
I believe it's because:
```go
// Deprecated: Core retrieves the default version of CoreClient.
// Please explicitly pick a version.
```